host/3.0/buster/amd64/python/python39/python39-composite.template

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9-slim-buster
+FROM mcr.microsoft.com/mirror/docker/library/python:3.9-slim-buster
```

Comment:
I don't like that appservice images rely on slim images but, non-appservice images (no suffix like Python39.Dockerfile) rely on fat images.

Appservice and nonsuffix images should be identical other than SSH tooling that we add to these images. 
I think that you should keep everything slim for the time being. I understand that its non-intuitive. sorry